packages/components/src/internal/url/URLResolver.ts

```diff
@@ -687,42 +700,41 @@ export class URLResolver {
     }
 
     // ToDo: this is rather fragile and data specific. this should be reworked with the mappers and rest of the resolvers to provide for more thorough coverage of our incoming URLs
-    resolveSearchUsingIndex(json): any {
-        let resolved = fromJS(JSON.parse(JSON.stringify(json)));
+    resolveSearchUsingIndex(result: SearchResult): SearchResult {
+        let resolved = fromJS(JSON.parse(JSON.stringify(result)));
 
         if (resolved.get('hits').count()) {
             const rows = resolved.get('hits').map(row => {
                 if (row && row.has('url')) {
-                    let url = row.get('url'),
-                        id = row.get('id'),
-                        column = List(); // no columns, so providing an empty List to the resolver
+                    const id = row.get('id');
+                    let url = row.get('url');
                     let query;
 
                     // TODO: add reroute for assays/runs when pages and URLs are decided
                     if (row.has('data') && row.hasIn(['data', 'dataClass'])) {
-                        query = row.getIn(['data', 'dataClass', 'name']); // dataClass is nested Map/Object inside of 'data' return
+                        query = row.getIn(['data', 'dataClass', 'name']); // dataClass is a nested Map/Object inside 'data' return
```

Comment:
"inside 'data' return"? Not sure what the "return" means here.

Reply:
Removed comment entirely as it doesn't really provide much insight.